google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/Util.java

```diff
@@ -108,20 +108,32 @@ static TagValue extractStatusFromFuture(Future<?> future) {
 
   static String extractTableId(Object request) {
     String tableName = null;
+    String authorizedViewName = null;
     if (request instanceof ReadRowsRequest) {
       tableName = ((ReadRowsRequest) request).getTableName();
+      authorizedViewName = ((ReadRowsRequest) request).getAuthorizedViewName();
     } else if (request instanceof MutateRowsRequest) {
       tableName = ((MutateRowsRequest) request).getTableName();
+      authorizedViewName = ((MutateRowsRequest) request).getAuthorizedViewName();
     } else if (request instanceof MutateRowRequest) {
       tableName = ((MutateRowRequest) request).getTableName();
+      authorizedViewName = ((MutateRowRequest) request).getAuthorizedViewName();
     } else if (request instanceof SampleRowKeysRequest) {
       tableName = ((SampleRowKeysRequest) request).getTableName();
+      authorizedViewName = ((SampleRowKeysRequest) request).getAuthorizedViewName();
     } else if (request instanceof CheckAndMutateRowRequest) {
       tableName = ((CheckAndMutateRowRequest) request).getTableName();
+      authorizedViewName = ((CheckAndMutateRowRequest) request).getAuthorizedViewName();
     } else if (request instanceof ReadModifyWriteRowRequest) {
       tableName = ((ReadModifyWriteRowRequest) request).getTableName();
+      authorizedViewName = ((ReadModifyWriteRowRequest) request).getAuthorizedViewName();
+    }
+    if (tableName == null) return "undefined";
```

Comment:
I think this condition should be if (tableName == null && authorizedViewName == null)?